mkdocs/docs/HPC/VScode.md

```diff
@@ -0,0 +1,46 @@
+# VS Code Tunnel
+
+Please keep in mind, that this is not a VS Code manual,
```

Comment:
use "`VS Code`" (with backticks, and consistent uppercase, see line below)

Reply:
maybe rename the md file to vscodetunnel, i think the url now would be like `.../vscode`. if this is only about the tunnel, make it more clear